hack/custom-boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The CDI Authors.
+Copyright YEAR The Kubernetes Authors.
```

Comment:
So I think this change is causing travis to fail.

Reply:
@awels Reverted that change.